### ISSUE TITLE
Is their a limit on how many clients can connect to this server? cause some clients can't connect.

### ISSUE THREAD

**Issue by smetronic** [BrandonPotter/SimpleTCP#50]
Hi, I have used this library in my IoT project in which the devices send the data to the server, the server application is running on windows server 2012. I can see all the devices which are connected to this server. Sometimes I have noticed when 20 devices are connected, other devices can't connect I have even tried to connect from my computer but it doesn't connect until I restart server application. This is becoming a big problem for me cause I am connecting more devices to the server and devices can't connect because of connection timeout.
-Thanks

**Comment by BrandonPotter** [comment]
20 should not be anywhere near the number of concurrent connections a TcpListener can accept.

Do you have a threadpool restriction? Each client connected will cause a read thread to spin up on the server...

**Comment by smetronic** [comment]
No, I don't think I have implemented something like that but I'm using dapper ORM to insert the data received from the clients.
